django/template/backends/jinja2.py

```diff
@@ -61,3 +65,29 @@ def render(self, context=None, request=None):
             context['csrf_input'] = csrf_input_lazy(request)
             context['csrf_token'] = csrf_token_lazy(request)
         return self.template.render(context)
+
+
+def get_exception_info(exception):
+    """
+    Formats exception information for display on the debug page.
+    """
+    context_lines = 10
```

Comment:
I guess the name/purpose of these variables are documented elsewhere, but maybe we could at least add a pointer to that. They aren't self-explanatory.